src/v2/Components/v2/ArtworkFilter/__tests__/ArtworkFilter.jest.tsx

```diff
@@ -63,16 +63,13 @@ describe("ArtworkFilter", () => {
       const wrapper = await getWrapper("lg", {
         onFilterClick,
       })
-      wrapper
-        .find("WaysToBuyFilter")
-        .find("Checkbox")
-        .first()
-        .simulate("click")
+      wrapper.find("WaysToBuyFilter").find("Checkbox").first().simulate("click")
 
       expect(onFilterClick).toHaveBeenCalledWith("acquireable", true, {
         acquireable: true,
         majorPeriods: [],
         page: 1,
+        sizes: [],
```

Comment:
This PR looks good! Would it be possible to add a test to cover state management around the array-type filters? (i.e. clicking "Small" and "Medium" correctly updates state and/or the UI).

It seems like it would be nice to have, but feel free to use your judgment if it's too complex.

Reply:
that's a good idea. I will try it in the morning.

Reply:
Added a nice test. 👍